python/paddle/trainer/config_parser.py

```diff
@@ -2457,13 +2457,11 @@ def __init__(
                 input_layer = self.get_input_layer(input_index)
                 operator_conf.input_sizes.append(input_layer.size)
                 operator_input_index.append(input_index)
-            if self.config.size == 0:
-                size = operator.calc_output_size(operator_conf.input_sizes)
-                if size != 0:
+            size = operator.calc_output_size(operator_conf.input_sizes)
```

Comment:
I see. That means the previous one is buggy. In the previous version line 2465 should be changed to 
sz = operator.calc_output_size()
if sz != 0: